lib/manageiq-network_discovery.rb

```diff
@@ -1 +1,19 @@
+module ManageIQ
+  module NetworkDiscovery
+    autoload :IpmiProbe, 'manageiq/network_discovery/modules/IpmiProbe'
+    autoload :MSScvmmProbe, 'manageiq/network_discovery/modules/MSScvmmProbe'
+    autoload :MSVirtualServerProbe, 'manageiq/network_discovery/modules/MSVirtualServerProbe'
+    autoload :RedHatRhevmProbe, 'manageiq/network_discovery/modules/RedHatRhevmProbe'
+    autoload :VMwareEsxVcProbe, 'manageiq/network_discovery/modules/VMwareEsxVcProbe'
+    autoload :VMwareServerProbe, 'manageiq/network_discovery/modules/VMwareServerProbe'
+    autoload :WindowsProbe, 'manageiq/network_discovery/modules/WindowsProbe'
+  end
+
+  module Providers
+    module Openstack
```

Comment:
I assume this is temporary for extraction purposes?

Reply:
Yes each autoload to go to their respective providers